config/routes.rb

```diff
@@ -7,4 +7,10 @@
   get "claim" => "claims#create"
 
   root to: "claims#new"
+
+  namespace :api do
+    namespace :v1 do
+      resources :claims, only: [:show]
```

Comment:
Does this allow passing in any "ID" to your new endpoint? For example, `GET /api/v1/claims/yesterday` would return _today's_ claims? That is potentially surprising? I'd opt for a non-standard action instead:

```ruby
resources :claims, only: [] do
  get :today
end
```